### ISSUE TITLE
themes: not complete or sufficiently hierarchial

### ISSUE THREAD

**Issue by davidhodge931** [StatisticsNZ/simplevis#243]
Currently, this does not work
```
library(simplevis)
library(palmerpenguins)
library(ggeasy)

p <- gg_point_col(penguins, 
             x_var = bill_length_mm, 
             y_var = body_mass_g, 
             col_var = species, 
             title = "Penguin size")

p +
  theme(text = ggplot2::element_text(colour = 'red')) + 
  theme(axis.text = ggplot2::element_text(colour = 'orange')) + 
  theme(title = ggplot2::element_text(colour = 'green'))
```
This is because elements need to be set up in a broad to narrow way with no args repeated.

For example:

Currently like this
```
        text = element_text(
          family = font_family,
          colour = "#323232",
          size = font_size_body
        ),
        strip.text = element_text(
          family = font_family,
          colour = "#323232",
          size = font_size_body,
          hjust = 0.475
        ),
        axis.title.x = element_text(
          family = font_family,
          colour = "#323232",
          size = font_size_body,
          margin = margin(t = 10)
        ),
```
Should be like this
```
text = element_text(
  family = font_family,
  colour = "#323232",
),
strip.text = element_text(
  size = font_size_body,
  hjust = 0.475
),
axis.title.x = element_text(
  size = font_size_body,
  margin = margin(t = 10)
),
```

**Comment by davidhodge931** [comment]
Test with `ggeasy`

```r
library(simplevis)
library(palmerpenguins)
library(ggeasy)

p <- gg_point_col(penguins, 
             x_var = bill_length_mm, 
             y_var = body_mass_g, 
             col_var = species)

p + easy_rotate_x_labels() 
p + easy_remove_legend() 
p + easy_x_axis_labels_size(22) 
p + easy_all_text_color("red", teach = T) # doesn't work
p + easy_remove_x_axis() 
p + easy_remove_y_axis(what = "ticks")
p + easy_move_legend("bottom")
p + easy_legend_at("left")
p + easy_rotate_legend("horizontal")
labelled::var_label(penguins$species) <- "Penguin\nSpecies"
p + easy_labs() # doesn't work
p + easy_rotate_y_labels(angle = "startatbottom", teach = TRUE)
```

**Comment by davidhodge931** [comment]
```r
library(ggplot2)
library(palmerpenguins)

ggplot(penguins) +
  geom_point(aes(bill_length_mm, body_mass_g, col = species)) +
  ggtitle("hello") +
  theme(text = element_text(colour = "red")) 

ggplot(penguins) +
  geom_point(aes(bill_length_mm, body_mass_g, col = species)) +
  ggtitle("hello") +
  theme(text = element_text(colour = "red")) +
  theme(axis.text = element_text(colour = "blue"))

ggplot(penguins) +
  geom_point(aes(bill_length_mm, body_mass_g, col = species)) +
  ggtitle("hello") +
  theme(text = element_text(colour = "red")) +
  theme(title = element_text(colour = "green")) +
  theme(axis.text = element_text(colour = "blue"))
```

**Comment by davidhodge931** [comment]
https://github.com/tidyverse/ggplot2/issues/4544

**Comment by davidhodge931** [comment]
Themes need to be complete with `complete = TRUE` argument to avoid inheriting from `theme_gray()`

**Comment by davidhodge931** [comment]
Fixed v4.4.1 :)
